doc/source/developer-guides/contributing.rst

```diff
@@ -45,6 +45,7 @@ The current code standards that are applied to any new changes:
 - **Tests**: We use pytest to run our tests that must continue to pass when new changes are integrated in the code. Regression tests, which are run by the continuous integration workflow are stored in ``qibo/tests``. These tests contain several examples about how to use Qibo.
 - **Coverage**: Test coverage should be maintained / be at least at the same level when new features are implemented.
 - **Pylint**: Test code for anomalies, such as bad coding practices, missing documentation, unused variables.
+- **Pre commit**: We use pre-commit to enforce automatation and to format the code.
```

Comment:
Do you mean
```suggestion
- **Pre commit**: We use pre-commit to enforce automation and to format the code.
```
?

Perhaps we could also say that this is enforced automatically for all PRs.

Reply:
Thanks for catching this.
Yes that's what I meant :)
Now that we've enabled the pre-commit action I should probably update this line.